test/sql.jl

```diff
@@ -1,3 +1,6 @@
+using Pkg
```

Comment:
The `Pkg.dir` here should be changed to use `@__DIR__`

Reply:
Fixed, thanks.